src/Extensions.Hosting.AsyncInitialization/Hosting/AsyncInitializationHostExtensions.cs

```diff
@@ -32,5 +32,35 @@ public static async Task InitAsync(this IHost host, CancellationToken cancellati
 
             await rootInitializer.InitializeAsync(cancellationToken);
         }
+
+
+        /// <summary>
+        /// Initializes and runs the application, by first calling all registered async initializers.
+        /// Upon completion, any registered initializers implementing <see cref="IAsyncTeardown"/> are called.
```

Comment:
This could be clearer. Upon reading this, my first reaction was "why would it invoke initializers after the host finishes running?".
Maybe something like this instead?
> After the host terminates, any registered initializers that implement <see cref="IAsyncTeardown"/> are called to perform the teardown.

Reply:
Done.